server/selection.go

```diff
@@ -162,6 +162,9 @@ func (s *MinLSSelector) selectUnknownSession() *BroadcastSession {
 	var addrs []ethcommon.Address
 	addrCount := make(map[ethcommon.Address]int)
 	for _, sess := range s.unknownSessions {
+		if sess.OrchestratorInfo == nil || sess.OrchestratorInfo.TicketParams == nil {
```

Comment:
Could this be shortened to `sess.OrchestratorInfo.GetTicketParams() == nil` since `GetTicketParams()` handles the case where `OrchestratorInfo` is nil gracefully?

Reply:
fixed in 334e281b